ci/common.rb

```diff
@@ -42,6 +42,26 @@ def travis_pr?
   !ENV['TRAVIS'].nil? && ENV['TRAVIS_EVENT_TYPE'] == 'pull_request'
 end
 
+BAD_CITIZENS = {
+  'couch' => 'couchdb',
+  'disk' => 'sysstat',
```

Comment:
You're right, but but I'm not sure that adding the overhead of a new test in the matric is worth it: the disk check doesn't require any install. https://github.com/DataDog/dd-agent/blob/master/tests/checks/integration/test_disk.py

It could event be part of `core_integration`: thoughts ?